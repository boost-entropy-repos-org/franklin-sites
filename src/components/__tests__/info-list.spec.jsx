import React from 'react';
import renderer from 'react-test-renderer';

import InfoList from '../info-list';

describe('InfoList component', () => {
  test('should render', () => {
    const infoData = [
      {
        title: 'Item 1',
        content: <div>Some content</div>,
      },
      {
        title: 'Another item',
        content: <div>Some more content</div>,
      },
    ];
    const component = renderer.create(<InfoList infoData={infoData} />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
