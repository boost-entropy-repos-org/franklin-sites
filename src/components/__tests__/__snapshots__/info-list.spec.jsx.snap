// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoList component should render 1`] = `
<div
  className="info-list"
>
  <div
    className="info-list__item"
  >
    <div
      className="info-list__item__title"
    >
      Item 1
    </div>
    <div
      className="info-list__item__content"
    >
      <div>
        Some content
      </div>
    </div>
  </div>
  <div
    className="info-list__item"
  >
    <div
      className="info-list__item__title"
    >
      Another item
    </div>
    <div
      className="info-list__item__content"
    >
      <div>
        Some more content
      </div>
    </div>
  </div>
</div>
`;
